src/index.html

```diff
@@ -7,6 +7,356 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="nav header__logo">
+        <a href="#home" class="nav__logo">
+          <img src="images/logo.png" alt="MOYO logo">
+        </a>
+      </div>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#laptops"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#gadgets" class="nav__link">
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#tablets" class="nav__link">
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#photo" class="nav__link">
+              Photo
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#video" class="nav__link">
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="cards-container">
+      <div class="card" data-qa="card">
+        <div class="card__image">
+          <img
+            src="images/imac.jpeg"
+            alt="APPLE A1419 iMac 27inch Retina 5K Monoblock (MNED2UA/A)"
+            class="card__product-image"
+          >
+        </div>
+        <div class="card__content">
+          <div class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</div>
+          <div class="card__code">Product code: 195434</div>
```

Comment:
```suggestion
          <div class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</div>
          
          <div class="card__code">Product code: 195434</div>
```

Reply:
Fix this everywhere
<img width="1085" alt="image" src="https://github.com/mate-academy/layout_catalog/assets/86789161/cabe93ce-91c6-48f9-b9fb-8b53bcc76984">